<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayMedia</class>
 <widget class="QDialog" name="PlayMedia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>player</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget
{
	border-image: url(:/my_image/IMG_0125.JPG);
}</string>
     </property>
     <widget class="QToolButton" name="rew_Button">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>310</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);
border-image: url(:/my_image/Player Fwd.ico);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="end_Button">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>310</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);
border-image: url(:/my_image/Player End.ico);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="play_pause_Button">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);
border-image: url(:/my_image/Player Pause.ico);
border-image: url(:/my_image/Player Play.ico);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回主界面</string>
      </property>
     </widget>
     <widget class="QToolButton" name="fwd_Button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>310</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);
border-image: url(:/my_image/Player Rew.ico);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="addsong_pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>添加歌曲</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>251</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
background-color: rgb(255, 220, 211);</string>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>37</number>
      </attribute>
      <column>
       <property name="text">
        <string>歌曲列表</string>
       </property>
      </column>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>311</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
    QSlider::add-page:Horizontal
     {     
background-color: rgba(255, 255, 255,100);
        height:4px;  
     } 
     QSlider::sub-page:Horizontal   
    {  
background-color: rgb(255, 255, 255);
        height:4px;
     }
    QSlider::groove:Horizontal   
    { 
        background:transparent; 
        height:4px;
    }  
    QSlider::handle:Horizontal  
    { 
        height:16px; 
        width:16px; 
	border-image: url(:/my_image/圆形 (9).png);
	
  margin: -6 0px; 
    }
      
</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>240</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/login/loging_button_bj.png);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>50</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/my_image/volume_32px_1138024_easyicon.net.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
